### ISSUE TITLE
Alt-clicking to create a tag page if non-existent

### ISSUE THREAD

**Issue by j-adel** [pjeby/tag-wrangler#65]
I'm absolutely loving the tag page addition! One problem though: alt-clicking a tag that doesn't have a tag page just defaults to the search functionality. It's hard for me to remember which tags have pages and which do not. It is also quite annoying having to create a tag page for every new note. So it'd be nice to create them automatically :D

Thanks so much for this awesome feature!!

**Comment by pjeby** [comment]
I'll have to give this some thought, but I could maybe see popping a confirmation prompt for creation if you alt-click a tag without a page.

**Comment by skyue** [comment]
I also need this feature. and it will make interaction more consistent.

**Comment by mariomui** [comment]
My only qualm about this is that it would be a default behavior and can't be toggled off by the user. Any creation/deletion without a visual user agency component might be irksome to new users.

**Comment by pm64** [comment]
True, but the behavior is consistent with Obsidian's handling of note links. So the user will likely have already been irked by that.

**Comment by j-maffe** [comment]
I also don't understand why one would want a tag page for just a select few of the tags and for the rest to default to the search bar (especially since the search bar sucks). That's just inconsistent behavior. I think the majority of users would like to have a tag page for every tag.

**Comment by pjeby** [comment]
The next version of Tag Wrangler (0.6.x) will pop up a confirmation dialog to create the tag if you alt (or option) click a tag, either in a tag pane or note.  If you cancel the confirmation, the normal search click will take effect

**Comment by mariomui** [comment]
@j-maffe I dont need an explanation of all the tags. I try to categorize manually as little as possible. I only need the tag page for the tags with subtags (and even then I'll try to limit the subtags.)
Subtags are like clades in paleo taxonomy. 
Tags don't last long in my world, plus it pollutes the notes with actual content. ( I link as i write) 

I only use dashboards for file management. My dash tabulates all the files in descending order of 'weakest' to 'strongest' note. If I had a page for every tag, the number of false positives or 'to-be-worked-on' notes would increase enormously by N (where N is the number of tags)

Not to mention that tags are usually very bad in the beginning and require doing a project to fully classify the disparate ideas. Like writing/worldbuilding/characters is better served with writing/sharedCharacter and writing/compendium-lists

**TLDR** I wouldn't want a tag page per tag. it would overwhelm me.

**Comment by pjeby** [comment]
Closing since this feature was released in version 0.6.0
